src/main/java/heronarts/lx/utils/LXUtils.java

```diff
@@ -499,4 +499,17 @@ public float get(double basis) {
   public static boolean isEmpty(String s) {
     return s == null || s.trim().isEmpty();
   }
+
+  /**
+   * Compare two objects either of which may be null.
+   */
+  public static boolean equals(Object obj1, Object obj2) {
```

Comment:
FYI this is `Objects.equals()` in standard Java
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-